<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Renamah</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabSimple">
       <attribute name="title">
        <string>Simple</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WidgetSimple" name="widgetSimple" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabLastOperations">
       <attribute name="title">
        <string>Last operations</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="FormLastOperations" name="widgetLastOperations" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoadProfile"/>
    <addaction name="actionSaveProfile"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionLanguage"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionLoadProfile">
   <property name="text">
    <string>&amp;Load profile...</string>
   </property>
  </action>
  <action name="actionSaveProfile">
   <property name="text">
    <string>&amp;Save profile...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormLastOperations</class>
   <extends>QWidget</extends>
   <header>form_last_operations.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetSimple</class>
   <extends>QWidget</extends>
   <header>widget_simple.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
